manifests/r/rocksdanister/LivelyWallpaper/1.7.2.0/rocksdanister.LivelyWallpaper.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: rocksdanister.LivelyWallpaper
+PackageVersion: 1.7.2.0
+PackageLocale: en-US
+Publisher: rocksdanister
+PublisherUrl: https://github.com/rocksdanister/lively
+PublisherSupportUrl: https://github.com/rocksdanister/lively/issues
+# PrivacyUrl: 
```

Do permalinks to files work for Privacy Policies?
```suggestion
PrivacyUrl: https://github.com/rocksdanister/lively/blob/ab73e0afb271e3b5c9f37f947ec75f1ceaf1f50a/PRIVACY.md
```